dev-tools/omdb/src/bin/omdb/nexus.rs

```diff
@@ -244,6 +248,36 @@ struct BlueprintImportArgs {
     input: Utf8PathBuf,
 }
 
+#[derive(Debug, Args)]
+struct SagasArgs {
+    #[command(subcommand)]
+    command: SagasCommands,
+}
+
+#[derive(Debug, Subcommand)]
+enum SagasCommands {
+    /// List sagas run by this Nexus
+    ///
+    /// Note that this is reporting in-memory state about sagas run by *this*
+    /// Nexus instance.  You'll get different answers if you ask different Nexus
+    /// instances.
```

Comment:
Actually, it occurs to me that we might want to consider giving this a less generic name than "list" to make it more obvious that it's *this* nexus' sagas only? Maybe `list-known` or something?

Reply:
That's #4378 by the way.

I left it at "list" because `omdb nexus` already implies a specific Nexus, but you're right that I obviously felt that wasn't sufficiently obvious.  A redundant name is fine if it's clearer but I don't think `list-known` is better (IMO).